lib/vector-core/src/transform/config.rs

```diff
@@ -15,10 +16,35 @@ pub enum ExpandType {
     Serial,
 }
 
+#[derive(Debug)]
+pub struct TransformContext {
+    pub globals: GlobalOptions,
+    pub enrichment_tables: EnrichmentTables,
```

Comment:
nit, riffing on the naming suggestion above.

```suggestion
    pub enrichments: enrichment::Database,
```

Reply:
Not so keen on the name Database. It's a bit of an overloaded term.

Reply:
Split the difference with `DataTable`

Reply:
I believe I suggested that at the start!

Reply:
I'm fine with other names, I don't think we should use the plural `Tables` though.

Reply:
It is now `TableRegistry` as per Lukes suggestion.